helm/hpcc/values.schema.json

```diff
@@ -355,6 +355,14 @@
             "storageWrites": {
                 "description": "cost per 10,000 write operations",
                 "type": "number"
+            },
+            "limit": {
+              "description": "Default cost limit for thor/hthor jobs (overrideable in ECL using maxCost option)",
+              "type": "number"
+            },
+            "hardlimit": {
```

Comment:
unrelated schema changes by the looks of it?

Reply:
I've removed the unrelated schema change and squashed with previous commit.